src/main/resources/db/changelog/changes/create-user-tenant-table.xml

```diff
@@ -29,7 +29,7 @@
       <column name="user_id"/>
     </createIndex>
     <createIndex indexName="username_tenant_id_idx" tableName="user_tenant" unique="true">
-      <column name="username"/>
+      <column name="user_id"/>
```

Comment:
we should create another changeset for doing this, in this new changeset we should delete old index and create new index

Reply:
because if you are modifiyng existing changes set - checksum validation will fail and so applying changes on envs where table was created previously also will fail

Reply:
fixed